freqtrade/exchange/binance.py

```diff
@@ -66,29 +107,104 @@ def stoploss(self, pair: str, amount: float, stop_price: float, order_types: Dic
 
             rate = self.price_to_precision(pair, rate)
 
-            order = self._api.create_order(symbol=pair, type=ordertype, side='sell',
+            order = self._api.create_order(symbol=pair, type=ordertype, side=side,
```

Comment:
I know i'm picky ... but (for futures, at least) you'll need to call `set_leverage` before this call (also in FTX)...